### ISSUE TITLE
Claims upload VoiceOver accessibility not reading

### ISSUE THREAD

**Issue by msmanak** [department-of-veterans-affairs/va.gov-team#20739]
## What happened?
With VoiceOver enabled on iOS, long text in claims upload page is not being announced but rather showing up as whole screen announcement

## Specs:
- Device: iOS iPhone X

## Steps to Reproduce

With VoiceOver enabled on iOS, navigate to claims in tool bar using staging account 366
click on first active claim
Click on "view file request"
scroll down to "request 2" to the text body

## Desired behavior
With VoiceOver enabled on iOS, the text body should be read out, not whole screen 

## Acceptance Criteria
See Desired behavior.

## Screen shot(s) and additional information
![IMG_723D7E0DC082-1.jpeg](https://images.zenhubusercontent.com/5faabf11b099434179a66d66/8cad58bf-d0e0-44ef-9473-64f393dcbf4d)

**Comment by bessgreen** [comment]
Could not reproduce this issue. 

Tested twice (@digitalken using an iPhone XR / @bessgreen using an iPhone 11), and we both observed the following behavior:

1. The long text is announced with VoiceOver, and it's grouped as 1 single, long announcement
2. When the user scrolls further down the page, the VoiceOver focus area overlaps with the header, but the header content isn't part of the announcement
